content/en/blog/_posts/2023-07-18-swap-beta1-graduation/index.md

```diff
@@ -0,0 +1,234 @@
+for a wide range of users. As a result, significant effort has been put into making
+the Beta version of Swap more stable, robust, user-friendly, and to address issues regarding the
+Alpha version that are mentioned above. This represents a crucial step towards achieving the
+goal of fully supporting swap in Kubernetes in a controlled, reliable, and predictable manner.
+
+## How do I use it?
+The utilization of swap memory on a node where it has already been provisioned can be
+facilitated by the activation of the `NodeSwap` feature gate on the kubelet.
+Additionally, the failSwapOn configuration setting must be disabled, or the
+`--fail-swap-on` command line flag must be deactivated.
+
+It is possible to configure the `memorySwap.swapBehavior` option to define the manner in which a node utilizes swap memory. For instance,
+
+```yaml
+memorySwap:
+  swapBehavior: UnlimitedSwap
+```
+
+The available configuration options for `swapBehavior` are:
+- `UnlimitedSwap` (default): Kubernetes workloads can use as much swap memory as they
+  request, up to the system limit.
+- `LimitedSwap` (default): The utilization of swap memory by Kubernetes workloads is subject to limitations. Only Pods of Burstable QoS are permitted to employ swap.
+
+If configuration for `memorySwap` is not specified and the feature gate is
+enabled, by default the kubelet will apply the same behaviour as the
+`UnlimitedSwap` setting.
+
+Note that `NodeSwap` is supported for **cgroup v2** only. cgroup v1 is no longer supported.
+
+### Install a Swap-enabled cluster with kubeadm
+
+#### Before you begin
+
+It is required that the kubeadm tool be installed, following the steps outlined in the
+[kubeadm installation guide](/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm).
+If swap is already enabled on the node, cluster creation may
+proceed. If swap is not enabled, please refer to the provided instructions for enabling swap.
+
+##### Create a swap file (4Gi here) and turn swap on
+
+```bash
+dd if=/dev/zero of=/swapfile bs=128M count=32
+chmod 600 /swapfile
+mkswap /swapfile
+swapon /swapfile
+swapon -s
+```
+
+To start the swap file at boot time, add line like `/swapfile swap swap defaults 0 0` to `/etc/fstab` file.
+
+
+#### Create a Swap-enabled cluster
+
+To make things clearer, here is an example kubeadm configuration file `kubeadm-config.yaml` for the swap enabled cluster.
+
+```yaml
+---
+apiVersion: "kubeadm.k8s.io/v1beta3"
+kind: InitConfiguration
+---
+apiVersion: kubelet.config.k8s.io/v1beta1
+kind: KubeletConfiguration
+failSwapOn: false
+featureGates:
+  NodeSwap: true
+memorySwap:
+  swapBehavior: LimitedSwap
+```
+
+Then create a cluster using `kubeadm init --config kubeadm-config.yaml`.
+During the init, there is a warning about swap is enabled on the node in case the kubelet `failSwapOn` is by default true, and we plan to remove it in future release.
```

Comment:
```suggestion
During the init, there is a warning that swap is enabled on the node in case the kubelet `failSwapOn` is by default true. We plan to remove this warning in a future release.
```